File: java/src/main/java/info/dmerej/Shop.java
package info.dmerej;

public class Shop {
    public static boolean canOrder(User user) {
        boolean minor = user.age() < 18;
        boolean verified = user.verified();

        return !minor;
    }

    public static boolean mustPayForeignFee(User user) {
        return !user.addres().country().equals("USA");
    }
}

File: cs/OnlineShop/Shop.cs
namespace OnlineShop;

public class Shop
{
  public static bool CanOrder(User user)
  {
    if (user.Age < 18)
    {
      return false;
    }

    if (!user.Verified)
    {
      return false;
    }

    return true;
  }

  public static bool MustPayForeignFee(User user)
  {
    if (user.Address.Country != "USA")
    {
      return true;
    }

    return false;
  }

}